v3/src/components/map/map-types.ts

```diff
@@ -46,7 +47,16 @@ export const
 
 
   kLatNames = ['latitude', 'lat', 'latitud', 'breitengrad', '緯度', 'عرض جغرافیایی'],
-  kLongNames = ['longitude', 'long', 'lng', 'lon', 'longitud', 'längengrad', '経度', 'طول جغرافیایی']
+  kLongNames = ['longitude', 'long', 'lng', 'lon', 'longitud', 'längengrad', '経度', 'طول جغرافیایی'],
+
+  // TODO: Figure out a better names for these in languages that don't use the latin alphabet.
```

Comment:
Actually, since these are user-visible strings, the word 'pin' should be translated for all the non-English names. That should at least be noted in the TODO.